data/wanderer ships.txt

```diff
@@ -855,3 +855,55 @@ ship "Deep River Transport"
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
 	description "This is a Deep River freighter that has been converted into a transport ship for refugees."
+
+
+
+ship "Riptide"
+	sprite "ship/riptide"
+	thumbnail "thumbnail/riptide"
+	attributes
+		category "Transport"
+		licenses
+			Wanderer
+		"cost" 21120000
+		"shields" 38600
+		"hull" 22350
+		"required crew" 47
+		"bunks" 233
+		"mass" 350
+		"drag" 5.1
+		"heat dissipation" .7
+		"fuel capacity" 800
+		"cargo space" 164
+		"outfit space" 488
+		"weapon capacity" 98
+		"engine capacity" 186
```

Comment:
I would recommend dropping the engine capacity down by 62 to 124, then remove one of the Type 4 steering and replace the Type 2 thruster with a type 1. 

This would drop the stock stats a notch to be a bit more reasonable:
Max speed: 856 down to 727
acceleration: 326 down to 300
Turning: 208 to 113